custom-nginx-reverse-proxy.conf

```diff
@@ -23,6 +23,16 @@ http {
             proxy_set_header   X-Forwarded-Host $server_name; 
         }
 
+        location /editor/ {
+            # editor is not dockerized, refer to host
+            proxy_pass         http://host.docker.internal:6666/;
```

Comment:
Shouldn't the port be 8081?

Reply:
No, `engine-and-editor` is in `:8081` so this one must be in some other port. `editor` is a completely new application (the extracted `editor` from `engine-and-editor`)